### PR TITLE
Bump chart and images

### PR DESCRIPTION
**Pull request by marcsanmi**
Closes https://github.com/newrelic/nri-kubernetes/issues/456.

### CODE REVIEW

File: charts/newrelic-infrastructure/values.yaml [hunk 1]
```diff
@@ -36,7 +36,7 @@ images:
   # @default -- See `values.yaml`
   integration:
```
Comment (gsanchezgavier) [inline charts/newrelic-infrastructure/values.yaml:37]: could you bump the infrastructure bundle and forwareder images?

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## 3.4.0
+
+### Added
+* Add k8s v1.23 & v1.24 new metrics:
```
[the remark this answers is not in the document]
Comment (marcsanmi) [reply CHANGELOG.md]: Done in c186747d4b0dcb487aa85c04755df4e2b3c32397.

File: charts/newrelic-infrastructure/values.yaml (hunk 1 above)
Comment (marcsanmi) [reply charts/newrelic-infrastructure/values.yaml:37]: Done in c186747d4b0dcb487aa85c04755df4e2b3c32397.

**Review by gsanchezgavier — APPROVED** [review]
LGTM

**Pull request merged**